airflow/dags/airtable_views/california_transit_organizations.sql

```diff
@@ -0,0 +1,34 @@
+---
+operator: operators.SqlToWarehouseOperator
+dst_table_name: "views.airtable_california_transit_organizations"
+
+description: Data from the organizations table in the California Transit Airtable base. See Airtable data documentation for more information about its contents. This table represents only the most recent extract and does not track changes in data over time.
+
+fields:
+  organization_id: Internal Airtable ID for this organization record, used for joining with other Airtable views tables
+  name: Organization name
+  organization_type: Categorical data; example values "City/Town", "Joint Powers Agency"
+  roles: Categorical choices imported as a string where a comma separates different selections; example values "Regional Transportation Planning Agency, Metropolitan Planning Organization"; "Metropolitan Planning Organization"
+  itp_id: Cal-ITP ID
+  details: Text description related to the organization
+
+tests:
+  check_null:
+    - organization_id
+    - name
+  check_unique:
+    - organization_id
+
+external_dependencies:
+  - airtable_loader: california_transit_organizations
+---
+-- roles is a multi-select field in airtable
+-- turn it into a string that's comma-delimited
+  SELECT
+    organization_id,
+    name,
+    organization_type,
+    REPLACE(REPLACE(REPLACE(roles, "'",""), "[", ""), "]", "") roles,
```

Comment:
Maybe this is fine for now, but another thought is that instead of turning this into something comma-delimited, perhaps there can be another table consisting of the organizations and their roles. Also, maybe this translation doesn't even need to occur if the unnesting is possible using SQL in BigQuery.

Reply:
These multi-response fields are why I did this gross string manipulation and the reason I did it is that I find unnested data very difficult to work with in BQ. I thought that for analysts it would be easier to do a query on `roles like "%MPO%"` than to deal with the unnested data. Making a separate table is ok too, that's just going pretty far down the denormalized route and I wasn't sure how much we wanted to commit to that?